exonum-java-binding/integration-tests/pom.xml

```diff
@@ -61,10 +63,29 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
       </plugin>
 
+      <plugin><!--Do not create Jar-->
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-jar</id>
+            <phase/>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin><!--Do not install Jar-->
+        <artifactId>maven-install-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-install</id>
+            <phase/>
```

Comment:
With ^ is this configuration needed?

Reply:
If yes (= there is some other warning), there is a more appropriate https://maven.apache.org/plugins/maven-install-plugin/install-mojo.html#skip

Reply:
yes, it's required because it tries to install unexisting jar.